const NotFoundError = (res) => {
    return res
        .status(404)
        .send("Esse dado n√£o foi encontrado no bando de dodos");
};

module.exports = {
    NotFoundError,
};
